From 6d2eccdfe2936113ff080c4b95971d0d339c697f Mon Sep 17 00:00:00 2001
From: Andrew Kelley <andrew@ziglang.org>
Date: Fri, 5 Sep 2025 19:45:07 -0700
Subject: [PATCH 094/133] std: update for new packed union rules
X-Developer-Signature: v=1; a=openpgp-sha256; l=649; i=xtex@astrafall.org;
 h=from:subject; bh=THftrIee2naXZe1uVungbFGgqpOC3vHKmfLYge8UteQ=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDJn78jiPb33dPtGh363RIS645qtc+oH+1xf4o24tTTDwK
 mkN6nrSMZGFQYyLwVJMkaXIsMGbVSedX3RZuSzMHFYmkCHSIg0MDAwMLAx8uYl5pUY6Rnqm2oZ6
 hoY6RjpGDFycAjDVx78zMlyYpPKX/9WsnN8+Xfc0pDVC/lpffrhbrebgm/kJPxQ/SF1jZNhxeGP
 GLekrmtzrBH9/MD3v8YLlxuf6R13XDR+djNy1uYMDAA==
X-Developer-Key: i=xtex@astrafall.org; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

(cherry picked from commit b074299124e1403d48d9904c6ab855b7c72df9e0)
---
 lib/std/meta.zig | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/lib/std/meta.zig b/lib/std/meta.zig
index 65b7d60c1854..019b0e3910c5 100644
--- a/lib/std/meta.zig
+++ b/lib/std/meta.zig
@@ -1193,13 +1193,6 @@ test hasUniqueRepresentation {
 
     try testing.expect(hasUniqueRepresentation(TestStruct6));
 
-    const TestUnion1 = packed union {
-        a: u32,
-        b: u16,
-    };
-
-    try testing.expect(!hasUniqueRepresentation(TestUnion1));
-
     const TestUnion2 = extern union {
         a: u32,
         b: u16,
-- 
2.52.0

